devtools/lightning-inspect.js

```diff
@@ -740,6 +743,10 @@ window.attachInspector = function({Element, ElementCore, Stage, Component, Eleme
         }
     });
 
+    Element.prototype.updateDebugFocusState = function() {
+        val(this, 'hasfinalfocus', this.hasFinalFocus(), false);
```

Comment:
Shouldn't it be `isfocused`?

Reply:
Using `hasFinalFocus` seemed to better match the way that developers would understand the focus; being able to see the value in the inspector helps the QAs on our project, and having the two teams be aligned helps us.

At some point we might also look to expose `hasFocus` (which is true for the whole chain of focus-containing elements) but that's more complex to implement, and not immediately needed.

The description for this pull request does still mention 'isfocused' but imho that's out of date.

Reply:
I would assume developers care much more for final element rather than whole chain which is an implementation detail that `hasFinalFocus` indirectly exposes and may lead to confusion. Focus chain could be exposed with more descriptive flag indicating chain implementation in the future if ever necessary.